server/service/integration_core_test.go

```diff
@@ -202,16 +202,49 @@ func (s *integrationTestSuite) TestUserEmailValidation() {
 }
 
 func (s *integrationTestSuite) TestUserPasswordLengthValidation() {
-	params := fleet.UserPayload{
+	t := s.T()
+
+	// test when creating a new user
+
+	newUPars := fleet.UserPayload{
 		Name:  ptr.String("user_invalid_email"),
 		Email: ptr.String("test@example.com"),
-		// This is 73 characters long
+		// // This is 73 characters long
```

Comment:
nit: double escape characters

Reply:
My intention was for this to be a "sub-comment" to the single-commented line 207, do you think that's helpful?

Reply:
@mostlikelee is this an acceptable pattern or should we just stick with single `//`s?

Reply:
i'd stick with a single `//` for consistency